Formula/openssl@1.1.rb

```diff
@@ -23,14 +23,9 @@ class OpensslAT11 < Formula
 
   keg_only :shadowed_by_macos, "macOS provides LibreSSL"
 
-  on_linux do
-    resource "cacert" do
-      # homepage "http://curl.haxx.se/docs/caextract.html"
-      url "https://curl.haxx.se/ca/cacert-2020-01-01.pem"
-      mirror "https://gist.githubusercontent.com/dawidd6/16d94180a019f31fd31bc679365387bc/raw/ef02c78b9d6427585d756528964d18a2b9e318f7/cacert-2020-01-01.pem"
-      sha256 "adf770dfd574a0d6026bfaa270cb6879b063957177a991d453ff1d302c02081f"
-    end
+  depends_on "cacert"
```

Comment:
I was thinking of making this a Linux- and old-macOS-only dependency, so that we can keep the keychain `post_install` in this formula. This way we can still do `depends_on "cacert"` in `gnutls`.

Reply:
gnutls does the exact same Keychain stuff. I'm not sure I follow how there wouldn't be a conditional in `gnutls` as well.

Reply:
It's not exactly the same:

https://github.com/Homebrew/homebrew-core/blob/c30079a98e07b0138f7cb2cce418cb8d394f1ae4/Formula/gnutls.rb#L104-L112

Reply:
Honestly, I see no reason why that can't apply to OpenSSL as well. It's a self-signed Kerberos certificate - not a SSL root certificate.